src/contexts/ItemsContext.jsx

```diff
@@ -0,0 +1,152 @@
+import React, { useContext } from "react";
+import PropTypes from "prop-types";
+
+// data in our store
+const items = [
+  {
+    id: "",
+    inventory: 100,
+    title: "Super Long Watch Name",
+    desc: "Series 6. Red",
+    longDesc:
+      "Available when you purchase any new iPhone, iPad, iPod Touch, Mac or Apple TV, 4.99/month after free trial.",
+    fullDesc: "",
+    color: [
+      ["Sky Blue", "#80BDE3"],
+      ["Rose Gold", "#B76E79"],
+      ["green", "#9FE2BF"],
+    ],
+    price: 359,
+    src: "assets/apple-watch-red.png",
+    promo: "",
```

Comment:
What is promo? Do we need it for mvp?

Reply:
Not needed. Can be removed. Has minimal impact since its not being used by any page at the moment.